src/services/services.js

```diff
@@ -1,16 +1,16 @@
 import axios from "axios";
 
-export const getProducts = async () => await axios.get("/api/products");
+export const getProducts = async () => axios.get("/api/products");
```

Comment:
Can you explain how is this working without using await inside the func.

Reply:
Wherever I am calling these api calls used async await